common/game_concepts/wc_magic_lifestyles_concepts.txt

```diff
@@ -0,0 +1,68 @@
+﻿wc_magic_lifestyle = {
```

Comment:
You can't choose the Focuses if you don't have Doctrine, even if you already have experience. Not sure if this is intended.

![bild](https://user-images.githubusercontent.com/4825809/114286422-db31a800-9a5e-11eb-9d68-c5b0e44fc948.png)

Reply:
Done!